### ISSUE TITLE
Collect Space Wrangler Bios

### ISSUE THREAD

**Issue by edrushka** [MozillaFoundation/Mozfest2016_production#56]
Need for each SW:
- photo
- Bio (we need a max word count)
- twitter handle
- blog (if applicable)

We can compile these on the [Github handles doc](https://docs.google.com/spreadsheets/d/100wmCld-YGysLHCpzYgovwctJkezPqQgFcOJ65mjyUk/edit#gid=0).

cc @Saallen @marcwalsh

**Comment by Saallen** [comment]
I have been thinking about the pictures and I think they should show a bit of personaility- MozFest is not like other festivals, its a place to come learn and play- I think this should be reflected in those photos- so No dog or squgily lines but also not corporate serious pics

**Comment by edrushka** [comment]
+1 to showing personality in photos. I think the key thing is consistency in sizing so it will look good in print. As long as the photo can be cropped to passport-style (i.e. face fills the frame), it's fine by me. The photos on the [Festival Team](https://mozillafestival.org/team) page are good examples  (except for mine - ha!)

**Comment by edrushka** [comment]
Here's the notes from the Festival Guide design brief:
- Space wrangler bios (there are 32)
- Name, Madlib: Does X at Y
- EX: Luca bridge youth and digital art at Tate Modern 
- No job title/roles -- descriptive instead.
- No handles, blog links etc. Just descriptive text
- Photos can be nice, clear headshots (like passport) (collect these tomorrow)

To Do: Write an example

**Comment by edrushka** [comment]
A few different bio styles:

**Previous job madlib.** Example from [LitHop 2016](https://lithopfresno.wordpress.com/writers-bios/):
- Paul Aloojian was born and raised in Fresno. He studied modern dance at Fresno City College, philosophy at UCLA, and earned a B.A. in English from California State University, Fresno in 1970. He has worked as an emergency mountain fire fighter, a grocery clerk, and a guest lecturer at UC Santa Cruz Extension. He is co-author of the anthology, Armenian Town. Event: Earning the Long View (3:00 p.m. at Spectrum Gallery)

**Tweet-length** Example from [SXSW](https://www.sxsw.com/speaker/mary-lou-jepsen/)
- Mary Lou Jepsen / SXSW Featured Speaker / Virtual Reality impresario leading a concerted effort to bring VR to the next level at Facebook/Oculus VR and formerly Google X.

**Localized** Example from [ubud writers festival](http://www.ubudwritersfestival.com/writers/emi-mahmoud/)
- Emi Mahmoud
  Nationality: Sudanese-American
  A poet and activist originating from the war-torn region of Darfur, Emi Mahmoud is the reigning Individual World Poetry Slam Champion of 2015. Currently a senior at Yale University, a Leonore Annenberg Scholar, and Global Health Fellow, Emi studies Anthropology and Molecular Biology at Yale in the hopes of one day alleviating structural disparities on maternal and child health in disadvantaged communities the world over.
  Supported by Embassy of the United States
  Emi Mahmoud berasal dari sebuah wilayah yang hancur oleh peperangan, Darfur. Ia adalah seorang penyair dan aktivis pemegang gelar Individual World Poetry Slam Champion 2015. Saat ini Emi tengah menyelesaikan tahun terakhirnya di Yale University, ia mempelajari Anthropology dan Molecular Biology dengan harapan pada suatu hari nanti ia dapat mengurangi perbedaan kesehatan ibu dan anak di masyarakat yang kurang beruntung di Indonesia.
  Didukung oleh Embassy of the United States

**Plain English, with character.** Example from [Rock and Roll writers festival](http://www.rockandrollwritersfestival.com/project/bc-michaels/)
- BC Michaels is the drummer & songwriter for Brisbane’s indie-garage-punk heroes Dune Rats. He’s been quoted as being on a “self-proclaimed quest to play music and fuck shit up” but in fact is a modern philosopher, as evidenced by the song “Dalai Lama, Big Banana, Marijuana’. However, he has zero friends on LinkedIn. Make of that what you will.

**Comment by edrushka** [comment]
My favourite are the bios from the Rock and Roll writers festival. I like the one-line 'official job title but in plain English'. We could follow that with something the person is interested in learning/doing/seeing at MozFest e.g.

ErikaD
I coordinate some of the programmatic bits and pieces at MozFest. 
As a tinfoil hat-wearing part-time technophobe, I want to know if there's a single 'thing' that's actually useful in the internet of things.

@marcwalsh - your thoughts on bio style?

**Comment by edrushka** [comment]
Incidentally, @Saallen, this site also features photos how I was trying to explain above: any style, but same cropping:
![screenshot 2016-08-16 17 02 57](https://cloud.githubusercontent.com/assets/4515662/17719989/62f784d4-63d3-11e6-9d0f-e96294759cd7.png)

**Comment by edrushka** [comment]
cc @KevZawacki

**Comment by KevZawacki** [comment]
@edrushka My vote:

Tweet-length Example from SXSW

Mary Lou Jepsen / SXSW Featured Speaker / Virtual Reality impresario leading a concerted effort to bring VR to the next level at Facebook/Oculus VR and formerly Google X.

**Comment by marcwalsh-zz** [comment]
@edrushka - My votes is the Rock and Roll Writers Festival style, it feels a lot more personal.

**Comment by edrushka** [comment]
👍 I'll draft an ask for Space Wranglers. We need to start gathering this content!

**Comment by edrushka** [comment]
Incoming bios are being collected on the second tab of [this sheet](https://docs.google.com/spreadsheets/d/100wmCld-YGysLHCpzYgovwctJkezPqQgFcOJ65mjyUk/edit#gid=1306798567).

**Comment by cubicgarden** [comment]
![ian-2016-wondering](https://cloud.githubusercontent.com/assets/1649922/18173993/bcebce86-7063-11e6-8347-ff01b1862a92.jpg)

Ian Forrester is a well known and likeable character on the digital scene in the UK. Living in Manchester, UK where he works for the BBC's R&D north lab. He specialises in open innovation and new disruptive opportunities; by creating value via open engagement and collaborations with startups, universities, early adopters and hackers.

@cubicgarden - Twitter
http://www.cubicgarden.com

**Comment by omnignorant** [comment]
![robertfriedman](https://cloud.githubusercontent.com/assets/6335874/18286717/a6586452-7439-11e6-8d21-1b9aaddd7ca9.jpg)

Robert Friedman believes that all people should have equal and open access to the networks, opportunities and resources they need to make their dreams come true; that the internet is a basic human right fundamental to making this vision a reality; and that universal web literacy is where to start.

**Comment by techkim** [comment]
Kim Wilkens is the Computer Science Initiative Coordinator for K-8 at St. Anne’s-Belfield School and the founder of Tech-Girls. She is passionate about transforming technology users into technology creators, collaborators and activists. Kim looks forward to embracing the possibilities and challenges of the future with you at MozFest!

@kimxtom
http://techkim.wikispaces.com
![kimpic14](https://cloud.githubusercontent.com/assets/4766034/18352340/0581e8ae-75ac-11e6-90b4-34b5d93d271b.jpg)

**Comment by simeonoriko** [comment]
![simeon oriko](https://cloud.githubusercontent.com/assets/389670/18352714/2e108190-75e8-11e6-8b5b-46a334306fb8.jpg)

Simeon Oriko, an Open Innovation strategist & consultant based in Nairobi, is passionate about helping people translate their ideas into reality. This passion shapes his vision and work at Jamlab, a co-creation community he established and serves as its Executive Director.

**Comment by gbaman** [comment]
Andrew Mulholland, a Computer Science student currently studying at Queens University in Belfast is passionate about getting kids excited and involved in Computer Science and Digital making. In his free time, he can be found working with schools and running events including the Northern Ireland Raspberry Jam.
Twitter - [@gbaman1](https://twitter.com/gbaman1)
http://blog.gbaman.info

![andrew-jamboree-high-res copy](https://cloud.githubusercontent.com/assets/833193/18480195/af14eb24-79cf-11e6-9ac7-cc2c681c291a.jpg)

**Comment by EPIKhub** [comment]
![foxdorine](https://cloud.githubusercontent.com/assets/10254892/18533228/52fe5978-7ad9-11e6-81ea-daff9ed9bee8.jpg)

**Comment by edrushka** [comment]
Hey @EPIKhub any chance you have a photo that's a little closer up? We need to crop them to head and shoulders to fit in the template that's been designed.

**Comment by EPIKhub** [comment]
@edrushka whats the latest I can get one to you?

**Comment by EPIKhub** [comment]
Also here is the Bio, weird doing 3rd person stuff:

"Heading up the "Youth led" programme at MozFest, her interest in Sociology and Anthropology led her to explore how digital culture can build resilience and inclusion across communities.

A keen RPG gamer, Dorine’s years spent assembling raid teams of young people in World of Warcraft lead her to a second career in HR supporting engineers (naturally)!"

**Comment by msaunby** [comment]
![saunby](https://cloud.githubusercontent.com/assets/848911/18554875/9a14c778-7b5d-11e6-88a0-ef4137ea169e.jpg)

Reinvented once again this former scientific software developer is currently teaching Python to degree apprentices, crafting IoT software and helping others deliver open innovation through hackathons and much more. 

@msaunby

**Comment by EPIKhub** [comment]
![dorine-jpp-7](https://cloud.githubusercontent.com/assets/10254892/18768520/a8297a78-811d-11e6-9d58-3d26a4ddbfcc.jpg)

This a better choice?

@dorineflies Twitter
@EPIKhub GitHub

**Comment by edrushka** [comment]
@carrie-ann, for the Festival Guide, let's call this final. Anyone who hasn't submitted their bios/photos yet can just be included as name only.

Final list content is [here in spreadsheet form](https://docs.google.com/spreadsheets/d/100wmCld-YGysLHCpzYgovwctJkezPqQgFcOJ65mjyUk/edit#gid=1306798567). You can also see the design approach we're using for the website [here on staging](https://mozillafestival-org-sta-pr-459.herokuapp.com/team/wranglers) -- you can use the same gear avatar for people with no photos if you want.

**Comment by EPIKhub** [comment]
Hi Erika,

Could you change the twitter account  I dont monitor @EPIKhub, can you
change it to @dorineFlies please?

With thanks
Dorine Flies

[image: --]
EPIK
[image: http://]about.me/dorine_flies
http://about.me/dorine_flies

**Comment by Carrie-ann** [comment]
@edrushka @kristinashu Thanks for these. Could someone please give me permission to view the document so I can grab the content and imagery. Thanks!

**Comment by kristinashu** [comment]
@Carrie-ann you should be able to view them now!

**Comment by Carrie-ann** [comment]
@kristinashu Thanks!

**Comment by edrushka** [comment]
Closing
